app/controllers/posts_controller.rb

```diff
@@ -4,7 +4,29 @@ def index
     @posts = Post.includes(:user).where(user: params[:user_id])
   end
 
+  def new
+    @post = Post.new
+  end
+
   def show
+    @user = User.find(params[:user_id])
     @post = Post.includes(:user, comments: [:user]).find(params[:id])
   end
+
```

Comment:
- [ ] Please implement **strong params** in creation of new posts, as informed in the previous code review. It should be similar to:
```ruby
  class PostsController < ApplicationController

    ...

    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end

    ...

  end
```

Reply:
Hi @rloterh, to create a post I implemented the strong params as seen on line 18 in PostsController. Can you please show me what else to implement? because I am stuck at the mo

Reply:
@ekenecf 
Hi sorry for the late response. I've been away for a while. Meanwhile  I can see that next reviewer explained the same to you.
Cheers, and happy coding!